src/store/state/state-model.ts

```diff
@@ -167,6 +167,7 @@ export const stateModel: StateModelIF = {
     feeAcknowledgement: false,
     naics: { ...EmptyNaics },
     businessType: null,
-    businessNumber: null
+    businessNumber: null,
+    businessTypeConfirm: false
```

Comment:
In the latest commit 5511d94, the initial value for businessTypeConfirm is now false for both SP and GP.

Reply:
Looks good